GQ_Characterize.cpp

```diff
@@ -84,7 +75,7 @@ GQ_Characterize::GQ_Characterize(double A, double B, double C, double D, double
   //use Moore-Penrose pseudoinverse to ensure minimal norm least squares solution
   //arma::mat Aai = pinv(Aa); //Original code; Working on the 3x3 Coefficient matrix
 
-  Eigen::Matrix3f Aai = pseudoInverse(Aa); // Aa.completeOrthogonalDecomposition().pseudoInverse();
+  Eigen::Matrix3f Aai = Aa.completeOrthogonalDecomposition().pseudoInverse();
```

Comment:
Was this here all along? Or a recent addition to Eigen?

Reply:
The [Eigen source code](https://bitbucket.org/eigen/eigen/annotate/default/Eigen/src/QR/CompleteOrthogonalDecomposition.h?at=default&fileviewer=file-view-default) indicates it has been around since early 2016.